tests/vmi_configuration_test.go

```diff
@@ -1687,8 +1687,14 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 
 			It("should add guest-to-memory headroom", func() {
 				const guestMemoryStr = "1024M"
-				origVmiWithoutHeadroom := libvmi.New(libvmi.WithResourceMemory(guestMemoryStr))
-				origVmiWithHeadroom := libvmi.New(libvmi.WithResourceMemory(guestMemoryStr))
+				origVmiWithoutHeadroom := libvmi.New(
+					libvmi.WithResourceMemory(guestMemoryStr),
+					libvmi.WithGuestMemory(guestMemoryStr),
```

Comment:
Otherwise this https://github.com/kubevirt/kubevirt/pull/12128/commits/1cce9be4c6c982def2bd08de9a3dbac670277e0a#diff-1d39a0c6d5edf649055941bb990db854c3807243c8577e2250b5c093c66b3964R1728 fails because the origVMI guest memory would be 0.